Tests/LeafErrorMiddlewareTests/DefaultLeafErrorMiddlewareTests.swift

```diff
@@ -143,57 +137,33 @@ class DefaultLeafErrorMiddlewareTests: XCTestCase {
         XCTAssertEqual(context.status, "401")
         XCTAssertEqual(context.statusMessage, "Unauthorized")
     }
-    
+
     func testNonAbort404IsCaughtCorrectly() throws {
         let response = try app.getResponse(to: "/404")
         XCTAssertEqual(response.status, .notFound)
         XCTAssertEqual(viewRenderer.leafPath, "404")
     }
-    
-    func testThatFuture404IsCaughtCorrectly() throws {
-        let response = try app.getResponse(to: "/future404")
-        XCTAssertEqual(response.status, .notFound)
-        XCTAssertEqual(viewRenderer.leafPath, "404")
-    }
-    
-    func testFutureNonAbort404IsCaughtCorrectly() throws {
-        let response = try app.getResponse(to: "/future404NoAbort")
-        XCTAssertEqual(response.status, .notFound)
-        XCTAssertEqual(viewRenderer.leafPath, "404")
-    }
-    
-    func testThatFuture403IsCaughtCorrectly() throws {
```

Comment:
Could you add a test to ensure that 403s are caught? I'm sure they are but it was a previous issue so just want to make sure we don't add any regressions in the future!